src/Traveler.js

```diff
@@ -6,8 +6,31 @@ class Traveler {
     this.tripsData = tripsData;
   }
 
-  filterTravelerTrips() {
-    return this.tripsData.filter(trip => trip.userID === this.id);
+  filterTravelerTrips(destinationData) {
+    let userTripsData = this.tripsData.filter(trip => trip.userID === this.id);
+
+    return userTripsData.map(trip => {
+      let destinationInfo = destinationData.find(destinationData => trip.destinationID === destinationData.id);
+
+      return {
+        id: trip.id,
+        travelers: trip.travelers,
+        date: trip.date,
+        duration: trip.duration,
+        status: trip.status,
+        suggestedActivities: trip.suggestedActivities,
+        destination: destinationInfo.destination,
+        totalTripCost: this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay).toFixed(2),
+        agencyFees: (this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay)*.10).toFixed(2),
+        overallCost: (this.calculateTotalTripCost(destinationInfo.estimatedFlightCostPerPerson, trip.travelers, trip.duration, destinationInfo.estimatedLodgingCostPerDay)*1.1).toFixed(2),
+        destinationImage: destinationInfo.image,
+        destinationAlt: destinationInfo.alt
+      }
+    })
+  }
+
+  calculateTotalTripCost(costPerPerson, totalTravelers, duration, lodgingCostPerDay) {
+    return (costPerPerson * totalTravelers) + (duration * lodgingCostPerDay)
```

Comment:
I like how you're adding all the trip and destination information into each trip. I think this will make it easier to populate the DOM with all the data by doing this up front. Looks good!